examples/BB84_cirq.py

```diff
@@ -0,0 +1,148 @@
+# Example program to demonstrate BB84 QKD Protocol
+
+################ Example output
+
+# 0: ────H───H───M───────
+
+# 1: ────X───M───────────
+
+# 2: ────X───H───M───────
+
+# 3: ────M───────────────
+
+# 4: ────H───M───────────
+
+# 5: ────H───H───M───────
+
+# 6: ────X───H───M───────
+
+# 7: ────X───H───M───────
+
+# 8: ────H───H───M───────
+
+# 9: ────M───────────────
+
+# 10: ───X───H───H───M───
+
+# 11: ───H───M───────────
+
+# 12: ───X───M───────────
+
+# 13: ───X───H───M───────
+
+# 14: ───M───────────────
+
+# 15: ───X───M───────────
+
+# 16: ───X───H───M───────
+
+# 17: ───X───M───────────
+
+# 18: ───H───M───────────
+
+# 19: ───H───M───────────
+
+# 20: ───H───M───────────
+
+# 21: ───H───M───────────
+
+# 22: ───X───H───M───────
+
+# 23: ───H───H───M───────
+# Simulating 5 repetitions...
+# Alice's Encoding basis: HCHCHHHHHCHCCCCCHCCHHCHH
+# Bob's Encoding basis:   HCCCCHCCHCHHCHCCCCHCCHCH
+# Alice's sent bits:      011000110010110111000010
+# Both bases Match::      XX_X_X__XXX_X_XX_X_____X
+# Expected key bits:      01_0_0__001_1_01_1_____0
+# Actual key bits:        010000110110
+
+################
+
+import numpy as np
+import cirq
+
+
+def main():
+
+    num_qubits = 24
```

Comment:
This is a lot of qubits for NISQ. Since we are going to be running this on Travis, let's use less resources.

Reply:
I've reduced it to 16 :)